ColabTool/Shared/PostFooter/index.jsx

```diff
@@ -58,13 +58,18 @@ const renderEdit = ({ onEditClick }) =>
 
 const renderApproval = ({
   onApproveClick,
-}) =>
-  <span className={style['post-button-last']}>
-    <span className={style['vertical-line']} />
-    <Button onClick={onApproveClick} noStyle>
-      <Text size={'small'} color={'blue'}>Approve</Text>
-    </Button>
-  </span>;
+  manager,
```

Comment:
i think this might be cleaner since there are only two states

```js
const renderApproval = ({
  onApproveClick,
  manager,
}) => ({
  manager ?
  <span className={style['post-button-last']}>
    <span className={style['vertical-line']} />
    <Button onClick={onApproveClick} noStyle>
      <Text size={'small'} color={'blue'}>Approve</Text>
    </Button>
  </span> :
  null
});
```

(there might be syntax errors here :smile:)